ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```diff
@@ -44,6 +44,140 @@ keys:
                   - str
                 default: 10000
                 description: Delay in milliseconds.
+      set_overload_bit:
+        type: dict
+        keys:
+          enabled:
+            type: bool
+          on_startup:
+            type: dict
+            keys:
+              delay:
+                type: int
+                convert_types:
+                  - str
+                description: Number of seconds.
+              wait_for_bgp:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                  timeout:
+                    type: int
+                    convert_types:
+                      - str
+                    description: Number of seconds.
+      authentication:
```

Comment:
Make sure to set fields as `required: true` if they are required to generate CLI.